lib/code_climate/test_reporter/client.rb

```diff
@@ -13,6 +13,11 @@ def host
         ENV["CODECLIMATE_API_HOST"] ||
           "https://codeclimate.com"
       end
+      
+      def ca_file
```

Comment:
I think `host` is called externally. I don't see this one needing to be called from outside the class, though